src/components/App/NavBar.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Navbar, DropdownButton, Dropdown, Form, FormControl, Button } from 'react-bootstrap';
+
+const NavBar = () => {
+  const loggedIn = useSelector((state) => {
+    return state.userReducer.loggedIn;
+  });
+
+  function ProfileButton(props) {
```

Comment:
rename to `profileButton` to name it like a function and remove the props argument